addons/io_hubs_addon/debugger.py

```diff
@@ -313,6 +359,9 @@ def draw(self, context: Context):
         row.operator(HubsUpdateSceneOperator.bl_idname,
                      text=f'{update_mode}')
 
+        row = box.row()
+        row.prop(context.scene.hubs_scene_debugger_room_export_prefs, "avatar_to_viewport")
```

Comment:
This seems to not do anything when spawning as an object (which is what I'd expect), so what do you think about greying it out?
```
if "debugLocalScene" not in hubs_session.room_params:
            row.active = False
```

Reply:
Good idea!. I'll add that.